src/dropdown/src/styles/index.cssr.ts

```diff
@@ -64,6 +66,16 @@ export default cB('dropdown-menu', `
           color: 'var(--option-text-color-active)'
         })
       ]),
+      cM('disabled', {
+        color: 'var(--text-color-disabled)',
+        cursor: 'not-allowed',
+        opacity: '.45',
```

Comment:
add `optionOpacityDisabled: opacityDisabled` to `light.ts`, then use `--option-opacity-disabled` rather than `.45`.

Reply:
just like select menu

Reply:
finished